### ISSUE TITLE
Identify SNAP contacts

### ISSUE THREAD

**Issue by edmullen** [18F/eligibility-rules-service#24]
Based on our WIC conversations, we have some leads on SNAP contacts. Let's settle on who and how to reach out.

**Comment by alexpandel** [comment]
I think we decided to wait on pursuing SNAP until after our follow-on conversations with WIC and CMS - is that right? Moving this to done for now, since we have a person identified to contact, and are just waiting on scheduling something with them until we decide that makes sense (which is already covered by #9).

**Comment by edmullen** [comment]
Yes, that's right.
